Game/Game.cs

```diff
@@ -18,7 +18,9 @@ public class Game : Node
 	public static bool WorldOpen = false;
 	public static StructureRootClass StructureRoot;
 
-	public static float MouseSensitivity = 1;
+	public static float LookSensitivity = 5;
+	public static float MouseDivisor = LookSensitivity/2;
```

Comment:
This should simply be
```cs
public static float MouseDivisor = LookSensitivity;
```
As it currently stands as soon as I merge this the default mouse sens would be doubled.
Say the mouse event outputs 10: 
 - Currently: `10 -> 10/(5/2) = 4 -> 4*5 = 20`
 - Should be: `10 -> 10/5 = 2 -> 2*5 = 10`